cmd/task/list.go

```diff
@@ -26,12 +25,20 @@ func List(server, taskView, pageToken string, pageSize uint32, all bool, writer
 
 	output := &tes.ListTasksResponse{}
 
+	state := tes.Unknown
+	if stateInt, ok := tes.State_value[stateFilter]; ok {
```

Comment:
The logic here should probably be a closer match to the `getTaskView` utility above so we get case insensitivity and an informative error message.